README.md

```diff
@@ -1,8 +1,14 @@
 # What is this working group about?
 
+![banner](~/Desktop/icon.png)
+
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+## Introduction
+
 _Growth, Maturity and Decline_ is a working group focused on the lifecycle of open source projects. We do not valence the stages, but seek to build awareness of those stages among consumers of these metrics. Decline can be ok when a project's utility is waning; or terrible if 20% of the entire internet relies on that project. Context matters, and we are not the arbiters of your context. The "valence" of "good/bad" emerges from use cases and YOUR context.
 
-This is a [CHAOSS](https://chaoss.community) working group, focused on the [Growth-Maturity-Decline][gmd] metrics category. Its aim is to coordinate the definition of the metrics, and the production of software for implementing them, in this area.
+This is a [wg-diversity-inclusion](https://chaoss.community) working group, focused on the [Growth-Maturity-Decline][gmd] metrics category. Its aim is to coordinate the definition of the metrics, and the production of software for implementing them, in this area.
```

Comment:
I think this is not right. This is still a CHAOSS working group, and certainly it is not the D&I working group ;-)

Reply:
Link to D&I: https://github.com/chaoss/wg-diversity-inclusion

Reply:
I think my comment still applies... This is the GMD working group ;-)